src/components/my-widgets-selected-instance.jsx

```diff
@@ -1,328 +1,423 @@
-											at
-											<span className="available_time">{ availability.end.time }</span>
-										</span>
-									: null
-								}
-
-								{availabilityMode == "anytime after"
-									? <span>
-											Anytime after
-											<span className="available_date">{ availability.start.date }</span>
-											at
-											<span className="available_time">{ availability.start.time }</span>
-										</span>
-									: null
-								}
-
-								{availabilityMode == "from"
-									? <span>
-											From
-											<span className="available_date">{ availability.start.date }</span>
-											at
-											<span className="available_time">{ availability.start.time }</span>
-											until
-											<span className="available_date">{ availability.end.date }</span>
-											at
-											<span className="available_time">{ availability.end.time }</span>
-										</span>
-									: null
-								}
-							</dd>
-							<dt>Access:</dt>
-							<dd
-								className={`access-level ${!inst.sharable || inst.is_draft ? 'disabled' : ''}`}
-								onClick={onPopup}
-							>
-								<span>
-									{inst.guest_access ? 'Guest Mode - No Login Required' : 'Staff and Students only'}
-								</span>
-
-							</dd>
-						</dl>
-						<a id="edit-availability-button"
-							role="button"
-							className={!inst.shareable || inst.is_draft ? 'disabled' : ''}
-							disabled={!inst.shareable || inst.is_draft}
-							onClick={onPopup}
-						>
-							Edit settings...
-						</a>
-					</div>
-
-				</div>
-
-				<div className={`share-widget-container closed ${inst.is_draft ? 'draft' : ''}`}>
-					<h3>
-						{inst.is_draft ? "Publish to share" : "Share"} with your students
-						<a href="https://ucfopen.github.io/Materia-Docs/create/assigning-widgets.html"
-							target="_blank">
-							View all sharing options.
-						</a>
-					</h3>
-					<input id="play_link"
-						type="text"
-						disabled={inst.is_draft}
-						readOnly
-						value={inst.play_url}
-					/>
-					<p>
-						Use this link to share with your students (or
-						<span
-							className="show-embed link"
-							onClick={() => {setShowEmbed(!showEmbed)}}
-						> use the embed code
-						</span>
-						).
-					</p>
-
-					{showEmbed
-						? <MyWidgetEmbedInfo inst={inst} />
-						: null
-					}
-
-				</div>
-			</div>
-			{showCopy
-				? <MyWidgetsCopyDialog onClose={() => {setShowCopy(false)}} onCopy={makeCopy} />
-				: null
-			}
-			{showCollab
-				? <MyWidgetsCollaborateDialog currentUser={currentUser} inst={inst} myPerms={myPerms} otherUserPerms={otherUserPerms} onClose={() => {setShowCollab(false)}} />
-				: null
-			}
-			{showExport
-				? <MyWidgetsExportDataDialog onClose={() => {setShowExport(false)}} />
-				: null
-			}
-			<MyWidgetsScores inst={inst} />
-		</section>
-	)
-}
-
-export default MyWidgetSelectedInstance
+import React, { useEffect, useRef, useCallback, useState, useMemo} from 'react'

```

Comment:
somethings up with the diff on this file.  First guess is the line endings changed (perhaps edited on Windows?)

I'll skip till the diff is more usefull